### PR TITLE
[TAS-1909] 💩 Do not send likerland verify email if not verified in authcore

### PR DESCRIPTION
**Pull request by williamchong**

### CODE REVIEW

File: src/server/api/routes/users/v2/email.js [hunk 1]
```diff
@@ -73,20 +74,22 @@ router.post('/email', authenticateV2Login, async (req, res, next) => {
...
-      subject: title,
-      title,
-      content: `<p>Please click the link to verify your email:</p><p>${verificationURL}</p>`,
-    });
-    await sendEmail({
-      email,
-      subject,
-      html: body,
-    });
+    if (verify !== '0') {
```
Comment (nwingt) [inline src/server/api/routes/users/v2/email.js:77]: Maybe add a comment here?

**Review by nwingt — APPROVED** [review]

File: src/server/api/routes/users/v2/email.js (hunk 1 above)
Comment (williamchong) [reply src/server/api/routes/users/v2/email.js:77]: about the meaning of `verify`, or when do we set `verify === '0'` ?
Maybe we should rename `verify` if its meaning is unclear

Comment (nwingt) [reply src/server/api/routes/users/v2/email.js:77]: How about this
```suggestion
    const hasVerifiedInAuthcore = verify !== '0';
    if (hasVerifiedInAuthcore) {
```

Comment (williamchong) [reply src/server/api/routes/users/v2/email.js]: Renamed var and added authcore description

**Pull request merged**